<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_cerar_empresa">
      <attribute name="title">
       <string>Crear Empresa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_crear_empresa">
         <property name="placeholderText">
          <string>Nombre de la Empresa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_crear_empresa">
         <property name="text">
          <string>Crear Empresa</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_editar_empresa">
      <attribute name="title">
       <string>Editar Empresa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget_editar_empresa"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_editar_empresa">
         <property name="text">
          <string>Editar datos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_eliminar_empresa">
      <attribute name="title">
       <string>Eliminar Empresa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget_eliminar_empresa"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_eliminar_empresa">
         <property name="text">
          <string>Eliminar empresa</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_ver_empresa">
      <attribute name="title">
       <string>Ver Empresa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget_ver_empresa"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
